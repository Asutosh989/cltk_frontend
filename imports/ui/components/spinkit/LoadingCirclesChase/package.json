{
  "name": "LoadingCirclesChase",
  "version": "0.0.1",
  "private": true,
  "main": "./LoadingCirclesChase.js"
}
