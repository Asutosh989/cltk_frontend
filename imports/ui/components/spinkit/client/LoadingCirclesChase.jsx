export default class LoadingCirclesChase extends React.Component {
	render() {
		return (
			<div className="loading-spinner loading-spinner-circles-chase">
				<div className="dot1"></div>
				<div className="dot2"></div>
			</div>
		);
	}
}
