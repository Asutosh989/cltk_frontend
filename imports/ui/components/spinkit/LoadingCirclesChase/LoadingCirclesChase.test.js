
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import LoadingCirclesChase from './LoadingCirclesChase';

describe('LoadingCirclesChase', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<LoadingCirclesChase />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
