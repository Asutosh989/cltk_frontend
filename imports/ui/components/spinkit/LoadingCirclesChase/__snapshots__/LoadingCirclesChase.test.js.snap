// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingCirclesChase renders correctly 1`] = `
<div
  className="loading-spinner loading-spinner-circles-chase"
>
  <div
    className="dot1"
  />
  <div
    className="dot2"
  />
</div>
`;
